__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+const {
```

Comment:
Instead of deconstruct all the different data, you can just call this something like 'testData' that way in your beforeEach instead of the object you can just have one variable called testData